### ISSUE TITLE
Wrong job status and running out of space

### ISSUE THREAD

**Issue by chrisgorgo** [OpenNeuroOrg/openneuro#483]
https://openneuro.org/datasets/ds001271/versions/00001?app=MRIQC-ICA&version=8&job=5aa938e9ba9c9a0001dc56ab

```
download failed: s3://openneuro.snapshots/b5ea7c3ba01c27b183312a9c2f017d9d/sub-1104/func/sub-1104_task-2step_run-03_bold.nii.gz to ../snapshot/data/sub-1104/func/sub-1104_task-2step_run-03_bold.nii.gz [Errno 28] No space left on device
download failed: s3://openneuro.snapshots/b5ea7c3ba01c27b183312a9c2f017d9d/sub-1104/func/sub-1104_task-2step_run-04_bold.nii.gz to ../snapshot/data/sub-1104/func/sub-1104_task-2step_run-04_bold.nii.gz [Errno 28] No space left on device
download failed: s3://openneuro.snapshots/b5ea7c3ba01c27b183312a9c2f017d9d/sub-1104/func/sub-1104_task-2step_run-05_bold.nii.gz to ../snapshot/data/sub-1104/func/sub-1104_task-2step_run-05_bold.nii.gz [Errno 28] No space left on device
download failed: s3://openneuro.snapshots/b5ea7c3ba01c27b183312a9c2f017d9d/sub-1104/func/sub-1104_task-rest_run-01_bold.nii.gz to ../snapshot/data/sub-1104/func/sub-1104_task-rest_run-01_bold.nii.gz [Errno 28] No space left on device
download failed: s3://openneuro.snapshots/b5ea7c3ba01c27b183312a9c2f017d9d/sub-1104/func/sub-1104_task-2step_run-06_bold.nii.gz to ../snapshot/data/sub-1104/func/sub-1104_task-2step_run-06_bold.nii.gz [Errno 28] No space left on device
The user-provided path /output/data does not exist.
```

yet the job was flagged as successful

(BTW this is a very large dataset 170Gb; 3Gb per participant)

**Comment by chrisgorgo** [comment]
Jobs that failed due to initial transfer disk space issue are still flagges as "successful" https://openneuro.org/datasets/ds001271/versions/00001/file-display/MRIQC%2Fdefault%2Fe8a4df62-357e-4f8c-8ce3-8408ec6b8593

**Comment by chrisgorgo** [comment]
This (incorrectly flagging failed jobs as successful) is still an issue. See https://openneuro.org/datasets/ds001271/versions/00001/file-display/MRIQC%2Fdefault%2F0460118e-a94f-48b6-bbf0-800b8bd56a83 "group" level)

**Comment by nellh** [comment]
Narrowed this down to flock not passing the exit code from aws cli back to Docker. Need to workaround this part of bids-app-host to fix this part of the issue.

**Comment by chrisgorgo** [comment]
This is still happening (openneuro/bids-app-host:0.8.4):
https://openneuro.org/datasets/ds001324/versions/00002/logs/FreeSurfer%2Fdefault%2Fd5f31861-51e0-4688-8ae9-be1a7ab88b94

**Comment by nellh** [comment]
Hi Chris, sorry I should have verified this was deployed after you asked earlier - it wasn't yet. 0.8.4 fixed #487, 0.8.5 fixes this issue. 0.8.5 is deployed now.
